internal/kafka/internal/workers/cluster_mgrs/dynamic_scaleup_mgr.go

```diff
@@ -172,6 +179,81 @@ func (m *DynamicScaleUpManager) processDynamicScaleUpReconcileEvent() error {
 	return nil
 }
 
+func (m *DynamicScaleUpManager) logReconcileEventEnd() {
+	glog.Infoln("dynamic scale up reconcile event finished")
+}
+
+// supportedInstanceTypeLocator is a data structure
+// that contains all the information to help locate a
+// supported instance type in a region's cluster
+type supportedInstanceTypeLocator struct {
+	provider         string
+	region           string
+	instanceTypeName string
+}
+
+func (l *supportedInstanceTypeLocator) Equal(other supportedInstanceTypeLocator) bool {
+	return l.provider == other.provider &&
+		l.region == other.region &&
+		l.instanceTypeName == other.instanceTypeName
+}
+
+// dynamicScaleUpExecutor is able to perform dynamic ScaleUp execution actions
+type dynamicScaleUpExecutor interface {
+	ScaleUp() error
```

Comment:
Is there a reason why these methods are exported?

Reply:
No reason in particular. I still not have a clear idea on what we will export.
I am guessing integration testing will influence this.
My preference in general is to have attributes/methods unexported if possible.

Reply:
+1 no issue for me with these being exportable for now until we know what we need to export. Thanks for the explanation! :)